src/utils/sformat.fs

```diff
@@ -1114,17 +1119,21 @@ namespace Microsoft.FSharp.Text.StructuredFormat
                               // massively reign in deep printing of properties 
                               let nDepth = depthLim/10
 #if FX_ATLEAST_PORTABLE
-                              System.Array.Sort((props),{ new System.Collections.Generic.IComparer<PropertyInfo> with member this.Compare(p1,p2) = compare (p1.Name) (p2.Name) } );
+                              System.Array.Sort((propsAndFields),{ new System.Collections.Generic.IComparer<MemberInfo> with member this.Compare(p1,p2) = compare (p1.Name) (p2.Name) } );
```

Comment:
we do an open System ... so I guess we can do Array.Sort here instead of namespace qualifying..